package/yast2-iscsi-lio-server.spec

```diff
@@ -17,7 +17,7 @@
 
 
 Name:           yast2-iscsi-lio-server
-Version:        3.1.19
+Version:        3.1.20
```

Comment:
as this go to master, it should bump version to 3.2.0 which is minor used in SP3